### DISCUSSION TITLE
can't run otlp_grpc_log example

### DISCUSSION THREAD

**Discussion by hamedprog** [open-telemetry/opentelemetry-cpp#1380] [Q&A]
i am compile opentelemetry 1.3 with cmake. using this option:

`cmake \
		-D BUILD_SHARED_LIBS=ON \
		-D BUILD_TESTING=OFF \
		-D CMAKE_POSITION_INDEPENDENT_CODE=ON \
		-D WITH_JAEGER=OFF \
		-D WITH_LOGS_PREVIEW=ON \
		-D WITH_OTLP=ON \
		.. \`
		
`cmake --build . --target all`
`cmake --install . --config Debug`
`ldconfig`

when I want to run example_otlp_grpc_log, run cant finish and goes to an infinite loop. CPU usage is high and the run can't be finished until I terminate it. whereas other examples run successfully. I used a debugger and found the line that causes a loop. 

problem is in this line:

`auto exporter    = std::unique_ptr<logs_sdk::LogExporter>(new otlp::OtlpGrpcLogExporter(opts));`

**Comment by esigo** [comment]
Hi @hamedprog, I'm not able to reproduce on Ubuntu latest. Also I think you are missing these two options in your `cmake` command:

``` cmake
-DWITH_EXAMPLES=ON
-DWITH_OTLP_GRPC=ON
```

**Reply by hamedprog** [reply]
rebuild with these new options.

same problem occurs

note that i install gRPC with `setup_grpc.sh` file
